### ISSUE TITLE
Building on macOS

### ISSUE THREAD

**Issue by Vutshi** [0ldsk00l/nestopia#426]
I have the following error when building nestopia 418ab507bd4fee74843e34970d13770f207b14df on macOS 15.0.1 (24A348):
```
source/fltkui/fltkui.cpp:592:14: error: 'update' is a protected member of 'Fl_Sys_Menu_Bar'
  592 |     menubar->update();
      |              ^
/usr/local/include/FL/Fl_Sys_Menu_Bar.H:59:8: note: declared protected here
   59 |   void update();
      |        ^
source/fltkui/fltkui.cpp:623:22: error: no member named 'about' in 'Fl_Sys_Menu_Bar'
  623 |     Fl_Sys_Menu_Bar::about(about, nullptr);
      |     ~~~~~~~~~~~~~~~~~^
2 errors generated.
make: *** [source/fltkui/nestopia-fltkui.o] Error 1
```

**Comment by rdanbrook** [comment]
On macOS you will need to use FLTK 1.4, which is not tagged yet, but I am hearing it is happening during October at some point. I'm not sure if they'll reach their goal of release this month, but FLTK 1.4 seems ready to me right now. Building and installing the pre-release version is fairly easy on macOS, and once you do that, things should work well for you. There are a lot improvements on macOS for the 1.4 release, it fixes a lot of glaring flaws 1.3 has on that platform.

**Comment by Vutshi** [comment]
Thank you @rdanbrook. 

After `brew install --HEAD fltk` I have successfully compiled `nestopia`.
